R/conversion.R

```diff
@@ -294,3 +316,18 @@ py_to_r.pandas.core.frame.DataFrame <- function(x) {
   df
   
 }
+
+#' @export
+summary.pandas.core.frame.DataFrame <- function(object, ...) {
+  summary.pandas.core.series.Series(object, ...)
+}
+
+#' @export
+length.pandas.core.frame.DataFrame <- function(x) {
+  length.pandas.core.series.Series(x)
```

Comment:
With an R `data.frame`, `length()` gives the number of columns. I think this would give the total number of elements?

Reply:
Oh wow I didn't know that before. Should I change this to match the data.frame behavior? Interestingly `length()` for an R matrix is the number of total elements.

Reply:
I think this should match the data.frame behavior.

For some context, this is because a data.frame is really just an R list under the hood, and so `length()` counts the number of elements in that list (non-recursively).

Matrices on the other hand are just R vectors with an extra dimension attribute, so length does give the total number of elements (pretending the matrix were just a long vector)

Reply:
Ok I'll change this. I doubt users will use length anyways. This is just to make sure the bahavior matches with data.frame. Users usually just use nrow, ncol, or dim as far as I know.